### DISCUSSION TITLE
Icon request: fa-payment-terminal (fa-pos-terminal)

### DISCUSSION THREAD

**Discussion by stijnvanbael** [FortAwesome/Font-Awesome#21184] [Icon Requests]
![products_920x690_weiss](https://cloud.githubusercontent.com/assets/1050148/9541346/7bad38d0-4d67-11e5-9dc1-1ccd1549b255.png)
![card_terminal-3-512](https://cloud.githubusercontent.com/assets/1050148/9541348/813f9f7c-4d67-11e5-9ec4-59cadd2fdb88.png)

**Comment by tagliala** [comment]
dup of #6792, please +1 that request and post there your examples

closing here

**Comment by stijnvanbael** [comment]
@tagliala it is not a duplicate. A [point of sale](https://en.wikipedia.org/wiki/Point_of_sale) refers to the whole setup where the customer pays. A [payment terminal](https://en.wikipedia.org/wiki/Payment_terminal) is more specific. It refers to the device that accepts electronic payments.

**Comment by jmadore** [comment]
I agree with stijnvanbael.
This is a different (but related) request.

This is a payment terminal, also know as a payment device, or POI (point of interaction.)
It is the actual unit with a pinpad where you swipe, dip (EMV/chip) or tap (contactless) your credit or debit card.

The other request is for the POS which referred to the PC, monitor/display, cash drawer and keyboard, 

![image](https://user-images.githubusercontent.com/7064917/30981637-0029c9b8-a453-11e7-933f-c4abeda2c02d.png)

![image](https://user-images.githubusercontent.com/7064917/30981684-1ac6e544-a453-11e7-8fa5-06501e6ae27f.png)

**Comment by olivierFr** [comment]
+1

**Comment by tagliala** [comment]
Sorry, of course this is not a duplicate.

Reopening here

**Comment by mitomm** [comment]
+1

**Comment by JhosenVinaNoa** [comment]
+1

**Comment by akScript** [comment]
+1

**Comment by saidbenmoumen** [comment]
+1

**Comment by codsane** [comment]
+1

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
